WordPress/wordpress-develop
# Fix: Misleading Message When Accessing Admin Menu Page

## Description

This PR addresses the issue where the misleading message `You do not have sufficient permissions to access this page.` was displayed when attempting to access an admin page that doesn't exist. Instead of suggesting a permissions problem, the fix ensures that users are now shown a clearer message: `The requested page does not exist.` This provides a more accurate and user-friendly explanation when the page is missing or unavailable.

## Screenshot

<img width="1466" alt="the-requested-page-does-not-exists" src="https://github.com/user-attachments/assets/583b2664-3d82-4df5-9487-5a4512099f9b">

## Testing Instructions

- Install a plugin that includes a settings page.
- Navigate to the plugin's settings page and copy the URL.
- Go to the plugins page and deactivate the plugin you just installed.
- Paste the copied URL of the plugin's settings page into the browser's address bar and press Enter.
- You should now see the message `The requested page does not exist.` instead of `You do not have sufficient permissions to access this page.`

## Trac ticket: https://core.trac.wordpress.org/ticket/14060
